src/sentry/event_manager.py

```diff
@@ -526,7 +526,11 @@ def stringify(f):
 
         ip_address = get_path(data, ['sentry.interfaces.User', 'ip_address'])
         if ip_address:
-            geo = geo_by_addr(ip_address)
+            try:
+                geo = geo_by_addr(ip_address)
+            except Exception:
+                geo = None
+
             if geo:
```

Comment:
It'd be cleaner now to just put this under an `else`

```python
try:
    geo = geo_by_addr(ip_address)
except Exception:
    pass
else:
    data['sentry.interfaces.User']['geo'] = {...}
```

Reply:
But then I'd still need the `if geo:` since it could be None, I didn't want to keep nested but I can if you prefer.

Reply:
Doesn't it just raise an exception always if it fails?

If not, then you can just keep it as is or whatever makes sense. I thought it always raised on any failure, but I guess that's not the case.